src/Renderer/AtlasBuilder.js

```diff
@@ -0,0 +1,94 @@
+import * as THREE from 'three';
+import Capabilities from '../Core/System/Capabilities';
+import fit from './Packer';
+
+export default function pack(images, uvs) {
```

Comment:
Can you document this function plz? (I struggle to understand it without it, and some comments below, maybe).